docs/mui-vale/styles/MUI/NoBritish.yml

```diff
@@ -1,5 +1,5 @@
 extends: substitution
-message: Use the US spelling '%s' instead of the British '%s'
+message: Write in the en-US locale '%s' instead of the en-UK one '%s'
```

Comment:
What's the reasoning for this change? The original message seems more clear to me.

Reply:
Ah ok, I felt that, e.g. British wasn't as clear as en-UK.

Reply:
```suggestion
message: Use the US spelling '%s' instead of the British '%s'
```